stackit/internal/services/argus/instance/resource.go

```diff
@@ -784,6 +1092,174 @@ func mapMetricsRetentionField(r *argus.GetMetricsStorageRetentionResponse, model
 	return nil
 }
 
+func mapUpdateAlertConfigField(ctx context.Context, resp *argus.UpdateAlertConfigsResponse, model *Model) error {
```

Comment:
This function and `mapAlertConfigField` only differ in the signature (because of the `resp` type) and in the first if block, right?

Reply:
If yes can't we encapsulate everything else after
```go
// Receivers
	respReceivers := *resp.Data.Receivers
```
in a sub function?